roles/tikv/templates/tikv.toml.j2

```diff
@@ -9,14 +9,17 @@
 {{ item }} = {{ value | to_json }}
 {% endfor %}
 
-[readpool.storage]
-{% for item, value in tikv_conf.readpool.storage | dictsort -%}
+[readpool]
```

Comment:
```suggestion
```

Reply:
There is a field `unify-read-pool` under the readpool section.

Reply:
If `[readpool]` is not deleted, the `[readpool]` item will be generated in the `tikv.toml`, but this configuration is not supported.

Reply:
I think it is supported in latest TiKV now.

Reply:
Reference to [#tikv.toml](https://github.com/tikv/tikv/blob/master/etc/config-template.toml), there is not `[readpool]` configuration.